test/functional/kernel/hyperplane/tests/test-kernel-hyperplane-3D.hpp

```diff
@@ -0,0 +1,172 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-23, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#ifndef __TEST_KERNEL_HYPERPLANE_3D_HPP__
+#define __TEST_KERNEL_HYPERPLANE_3D_HPP__
+
+#include <numeric>
+#include <type_traits>
+
+template <typename INDEX_TYPE, typename DATA_TYPE, typename WORKING_RES, typename EXEC_POLICY, typename REDUCE_POLICY>
+typename std::enable_if<std::is_unsigned<RAJA::strip_index_type_t<INDEX_TYPE>>::value>::type
+KernelHyperplane3DTestImpl(const int RAJA_UNUSED_ARG(groups), const int RAJA_UNUSED_ARG(idim), const int RAJA_UNUSED_ARG(jdim), const int RAJA_UNUSED_ARG(kdim))
+{
+  // do nothing for unsigned index types
+}
+
+template <typename INDEX_TYPE, typename DATA_TYPE, typename WORKING_RES, typename EXEC_POLICY, typename REDUCE_POLICY>
+typename std::enable_if<std::is_signed<RAJA::strip_index_type_t<INDEX_TYPE>>::value>::type
+KernelHyperplane3DTestImpl(const int groups, const int idimin, const int jdimin, const int kdimin)
+{
+  // This test traverses "groups" number of 3D arrays, and modifies values in a 2D hyperplane manner.
+
+  int idim, jdim, kdim;
+  if ( std::is_same<DATA_TYPE, float>::value )
```

Comment:
this test is sensitive too?

Reply:
Yes, it is sensitive to `float` as well. It's a sum, so that jives with the other sensitive sums in the kernel tests.